operator/controller/src/main/java/io/apicurio/registry/operator/resource/app/AppDeploymentResource.java

```diff
@@ -87,6 +88,9 @@ protected Deployment desired(ApicurioRegistry3 primary, Context<ApicurioRegistry
         // Configure the CORS_ALLOWED_ORIGINS env var based on the ingress host
         Cors.configureAllowedOrigins(primary, envVars);
 
+        // Configure the TLS env vars
+        TLS.configureTLS(primary, deployment, REGISTRY_APP_CONTAINER_NAME, envVars);
```

Comment:
I think TLS only makes sense for Registry container at the moment, so `REGISTRY_APP_CONTAINER_NAME` could be inlined. Optional suggestion, might be done in a separate PR, since it's also done elsewhere.

Reply:
The same value is also used in line 114 in this class, so I think it makes more sense to import the constant (easier refactor later if needed).